specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json

```diff
@@ -93,40 +93,6 @@
                     "nextLinkName": "nextLink"
                 },
                 "x-ms-odata": "#/definitions/FabricLocation"
-            },
-            "put": {
-                "x-ms-examples": {
-                    "Creates or updates a fabric location.  This will fail if called outside deployment.": {
-                        "$ref": "./examples/FabricLocation/Put.json"
```

Comment:
The example file will need to be deleted to make Avocado happy

Reply:
Agreed with @nschonni.

@cednation Please remove the example file too. 

Also, Please be aware this is a breaking change and thus needs ARM review

Reply:
done